client/my-sites/site-settings/form-jetpack-monitor.jsx

```diff
@@ -99,22 +101,45 @@ module.exports = protectForm( React.createClass( {
 		return (
 			<div>
 				<p>{ this.translate( "Jetpack is currently monitoring your site's uptime." ) }</p>
+				{ this.settingsMonitorEmailCheckbox() }
+				{ config.isEnabled( 'settings/security/monitor/wp-note' ) ? this.settingsMonitorWpNoteCheckbox() : '' }
```

Comment:
I think this could be replaced with { config.isEnabled( 'settings/security/monitor/wp-note' ) && this.settingsMonitorWpNoteCheckbox()  }